jenkinsfiles/cordova_jenkinsfile.groovy

```diff
@@ -41,22 +55,31 @@ node(platform) {
             } else {
                 println('Debug Build - Using default developer signing key')
             }
-        }    
+        }
+        if (platform == 'ios') {
+            codeSign(
+                profileId: '${CODE_SIGN_PROFILE_ID}',
+                clean: CLEAN,
+                verify: true,
+                ipaName: "${OUTPUT_FILE_NAME}",
+                appPath: "build/${BUILD_CONFIG}-${SDK}/${PROJECT_NAME}.app"
```

Comment:
for some reason cordova uses "emulator" instead of `${BUILD_CONFIG}-${SDK}` (for both appPath and archive)